### PR TITLE
Add installation and update functionality (Windows)

### PR DESCRIPTION
**Pull request by knuton**
Hallelujah! :shipit: 

This adds build steps for creating an installer for Windows and releasing the installer and delta updates to S3 at `dist.dividat.ch`.

### Summary

- Release script does some basic sanity checks to prevent bad releases.
- Releases are uploaded to `dist.dividat.ch/releases/driver/stable/win32/ia32` and `dist.dividat.ch/releases/stable/darwin`.
- The general form of the release path is `/releases/<product>/<channel>/<platform>(/<arch>)`.
- Redirects to latest installers/packages are at `/releases/<product>/<channel>/<platform>(/<arch>)/latest`, to be used from website etc.
- Currently the only channel is _stable_.
- If the Windows installer can find an existing Chrome installation, it creates shortcuts to launch Play in kiosk mode on startup and from the desktop. The user can delete those if undesired, updates won't restore them.
- The Windows build checks for updates on launch, and then every 12 hours.

### Out of Scope

- Package knowledge of the release channel into the application so that it checks the appropriate channel.
- The Mac version is a simple ZIP of the `.app` folder and does not do automatic updates yet.

### Regrets

- Trying to use GitHub for hosting releases turned out to be a huge mistake:
  - Support in Squirrel.Windows exists only to a degree, using GitHub on non-C# projects requires extra work in update checks.
  - All assets need to be attached to every release.
  - The GitHub API for release assets seemed to be unreliable at times: Uploads failed randomly, or worse, succeeded on the surface, but didn't really put the assets in storage.
- Testing and debugging Windows applications from Mac OS is rough.

### CODE REVIEW

File: bin/release [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+        distDir: 'build/dist'
+    }
+).catch(
+    (err) => { console.error(err); process.exit(1); }
+);
+
+async function release(options) {
+    prompt.start();
+
+    assert.equal(
```
Comment (pukkamustard) [inline bin/release]: Beautiful! 😍

File: Readme.md [hunk 2]
```diff
@@ -23,9 +32,13 @@ To start with Electron (as in production) run `npm run electron`.
 
 This will start the driver as an Electron application. Note that no window will be created. The application lives in the menubar.
 
-To build Electron releases: `npm run build`
+## Building Releases
+
+To build Electron executables and installers: `npm run build`
 
-# SSL
+Additional dependencies are required for creating the installers. If you use a Unix system, you will need to [install Mono](http://www.mono-project.com/download/) in order to create and sign installers for Windows.
```
Comment (pukkamustard) [inline Readme.md]: This is not yet the case, right? Currently installer is not being signed?

Move this note to "Releasing New Versions" section with a Todo note about signing the installer.

File: bin/release (hunk 1 above)
Comment (pukkamustard) [inline bin/release]: The `assert`...

File: Readme.md (hunk 2 above)
Comment (knuton) [inline Readme.md]: No, Squirrel also uses Mono, the build script as-is won't run without Mono.

**Comment by pukkamustard** [comment]
Hallelujah! 🙏 🎉

**Pull request merged**